integration/src/test/java/dev/ikm/tinkar/integration/langext/fhir/FhirTransformAPIIT.java

```diff
@@ -16,30 +16,17 @@
 package dev.ikm.tinkar.integration.langext.fhir;
 
 import dev.ikm.tinkar.common.id.IntIds;
-import dev.ikm.tinkar.coordinate.stamp.StampCoordinateRecord;
-import dev.ikm.tinkar.coordinate.stamp.StampCoordinateRecordBuilder;
-import dev.ikm.tinkar.coordinate.stamp.StampPositionRecord;
-import dev.ikm.tinkar.coordinate.stamp.StampPositionRecordBuilder;
-import dev.ikm.tinkar.coordinate.stamp.StateSet;
+import dev.ikm.tinkar.coordinate.stamp.*;
```

Comment:
We should avoid using * in import statements

Reply:
@dettmoney, thank you for the 'offline' suggestion about increasing the import * threshold for the IntelliJ Optimize Import settings.